### PR TITLE
Configuration for nb, nn

### PR DESCRIPTION
**Pull request by jhsoby**
Please let me know if there's something else that's needed to set up this tool for those languages.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -410,6 +410,72 @@
...
+        wikilink_prefix_blacklist = EN_WIKILINK_PREFIX_BLACKLIST,
+        tags_blacklist = EN_TAGS_BLACKLIST,
+        templates_blacklist = EN_TEMPLATES_BLACKLIST,
+        hidden_category = 'Skjulte kategorier',
+        category_name_regexps_blacklist = [
+            '.*[0-9]+.*',
+        ],
+    ),
+
+    nn = dict(
```
Comment (eggpi) [inline config.py]: It looks like we [don't have a translation](https://translatewiki.net/w/i.php?title=Special:Translate&filter=%21translated&group=citationhunt&task=custom&language=nn&action=translate) for this one yet, right? Can you take care of that please? I did manage to try out the config locally, it it looks mostly fine.

However, I ended up with only 579 snippets. This is not a problem in itself, but maybe there are other templates, or alternate spellings of the existing templates, that can be added to `citation_needed_templates`? Sorry these have to be specified explicitly for now, I haven't got around to working on #51 yet :(

File: config.py [hunk 2]
```diff
@@ -410,6 +410,72 @@
...
+        lang_dir = 'ltr',
+        database = 'nowiki_p',
+        wikipedia_domain = 'no.wikipedia.org',
+
+        beginners_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder',
+        beginners_link_title = 'Wikipedia:Bruk av kilder',
+        lead_section_policy_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder#Hvorfor_siterer_vi_kilder',
+        lead_section_policy_link_title = 'Wikipedia:Bruk_av_kilder',
+
+        citation_needed_category = 'Artikler som trenger referanser',
```
Comment (eggpi) [inline config.py]: Category names need to use underscores instead of spaces: `Artikler_som_trenger_referanser`, please.

File: config.py [hunk 3]
```diff
@@ -410,6 +410,72 @@
...
+        citation_needed_templates = EN_CITATION_NEEDED_TEMPLATES + [
+            'Trenger referanse',
+            'Tr',
+            'Referanse',
+        ],
+        citation_needed_template_name = 'trenger referanse',
+        wikilink_prefix_blacklist = EN_WIKILINK_PREFIX_BLACKLIST,
+        tags_blacklist = EN_TAGS_BLACKLIST,
+        templates_blacklist = EN_TEMPLATES_BLACKLIST,
+        hidden_category = 'Skjulte kategorier',
```
Comment (eggpi) [inline config.py]: Underscores here and also in the same places for nn, please.

File: static/style.css [hunk 4]
```diff
@@ -106,30 +106,44 @@ button:focus {
     margin-bottom: 40px;
 }
 
+
+
```
Comment (eggpi) [inline static/style.css]: No need for these blank lines, right?

File: static/style.css [hunk 5]
```diff
@@ -106,30 +106,44 @@ button:focus {
...
 .button {
-    margin: 0 5px;
-    padding: 10px;
-    border: 2px solid black;
-    border-radius: 4px;
-    text-decoration: none;
-    color: black;
-    font-weight: bold;
+    font-size:1em;
```
Comment (eggpi) [inline static/style.css]: Nitpick: please add spaces after the : for consistency with the rest of the file.

File: static/style.css (hunk 4 above)
Comment (jhsoby) [inline static/style.css]: No, those were a mistake I didn't notice until I commited. I'm still quite a newbie at this Git thing, so  I'm not sure how to fix that after I commited.

File: config.py (hunk 1 above)
Comment (jhsoby) [inline config.py]: 579 doesn't sound wrong actually, it's a pretty small wiki. They have ~2000 articles in the category for missing references, but most of those are via a general template saying that the _entire article_ is missing references.

File: static/style.css (hunk 5 above)
Comment (jhsoby) [inline static/style.css]: 👍

**Comment by eggpi** [comment]
Great, merged this as 7c5c6340d1cec070db56d34679156dbe9f206d34! I'll enable nb on Tools Labs, but we'll still need to wait for the nn translation to be committed to this repository by the Translate Wiki people. Thank you!

Oh and by the way, if you're having trouble with git, you may be interested in learning about `git rebase`, which lets you make changes to your history after committing. [This](https://www.atlassian.com/git/tutorials/rewriting-history) seems to be a good tutorial. I use it a lot on pull requests like this: make a change, use `git rebase` to clean up the branch, then `git push -f` to update it.

**Comment by eggpi** [comment]
Hey, these are now live!

https://tools.wmflabs.org/citationhunt/nn
https://tools.wmflabs.org/citationhunt/nb

By the way, it's worth mentioning that most of the traffic for English CH comes from links spread out in relevant pages on that Wikipedia (e.g, in https://en.wikipedia.org/wiki/Wikipedia:Citation_needed). It might make sense to do something like that here, too.

Anyway, thanks again for this!

**Pull request closed without merging**